browser/help/topics/v4-browser-hts.md

```diff
@@ -0,0 +1,223 @@
+---
+id: v4-browser-hts
+title: 'gnomAD v4 Browser Hail Tables'
+---
+
+To go from the gnomAD release tables to a format that is more convenient for the browser to display, the input tables undergo a few transformations. This groups data in ways that are more convenient to query from the database these tables ultimately get loaded into. In addition to performing some reshaping of the data, additional annotations or information is added as needed.
```

Comment:
I wonder if we can we add a bit more high level explanation about these tables? How about:

---

We release two additional data tables to the public so that our users can more easily incorporate gnomAD data into external pipelines and analyses in a manner consistent with what they see in the browser.

## gnomAD v4.1 exome/genome variant table

To convert the standard gnomAD variant release tables into a format more suitable for browser display, we join the exome and genome tables on locus/allele to create a single table. This process ensures that they share the same site-level annotations, thus saving space and optimizing database/API queries. Additionally, frequencies are structured in a JSON-like format that is more easily consumable by web applications.

Each row (i.e., variant) in this table will have distinct allele frequency information and quality metrics depending whether it was present in the exome or genome callsets, but will share common annotations such as [VEP annotations](https://useast.ensembl.org/info/docs/tools/vep/index.html) and _in silico_ predictors.

The script for how this table is created can be found [here](
https://github.com/broadinstitute/gnomad-browser/blob/main/data-pipeline/src/data_pipeline/pipelines/gnomad_v4_variants.py).

## gnomAD 4.1 genes table

The second table underlies the gene models data seen in the browser, which contains detailed information on exon-coding regions, transcripts, identifiers, gene constraint, and co-occurrence data. The data from this table is derived from [GENCODE 39](https://www.gencodegenes.org/human/release_39.html), the [HUGO Gene Nomenclature Committee (HGNC)](https://www.genenames.org/), [MANE transcripts](https://www.ncbi.nlm.nih.gov/refseq/MANE/), [GTEx](https://gtexportal.org/home/) (coming soon), and from gnomAD secondary analyses.

The script for how this table is created can be found [here](https://github.com/broadinstitute/gnomad-browser/blob/main/data-pipeline/src/data_pipeline/pipelines/genes.py).